rapid7/metasploit-framework
# Aruba testing

Adding Cucumber testing to metasploit framework
## Caution

this adds two new rake tasks, features:boot moves your database.yml to database.yml.local
This should not effect anything in framework its just a new test suite for travis-ci, its also a little fragil.
## Verification steps
- [ ] `rake features` this should have nothing but green output
- [ ] `rake features:boot` this should also pass

This is adding a very limited UI testing for msfconsole
This tests
- that we can load a module
- that ms08-067 has a given set of options
- that ms08-067 has a given set of advanced options
- that the help command has output
- that we can load msfconsole without a database.yml
- that we can load msfconsole with a database.yml that points to a database that isnt there

These tests are fragil and not the best solution but it should work for a limited set of functionality.
